refm/api/src/_builtin/Numeric

```diff
@@ -624,10 +649,12 @@ Numeric のサブクラスは、このメソッドを適切に再定義しなけ
 
 @param other 自身を割る数を指定します。
 
-  1.fdiv(3)            #=> 0.3333333333333333
+#@samplecode 例
+1.fdiv(3)            #=> 0.3333333333333333
 #@since 1.9.1
```

Comment:
この `#@since 1.9.1` (と対応する `#@end`) は消してしまって良さそうです。

Reply:
削除する意図としては『 1.9.1 はサポート外なのでもう削除してしまってよい』という感じでしょうか。
[他の `#@since 1.9.1` の部分](https://github.com/rurema/doctree/blob/78ab3946ac8281a8421e41592e95ff436015970f/refm/api/src/_builtin/Numeric#L643) も削除するべきなのかどうか悩んでいます。

Reply:
たとえば https://github.com/rurema/doctree/pull/2518 のように古いバージョンの対応は削除しても構わないです。
コミットはわかれている方が良さそうですが、 pull request 自体は他の変更と一緒でも良いと思います。

Reply:
https://github.com/rurema/doctree/pull/2528/commits/d1c3cf11db64efce2e8e689e9723eb71dffaeb59 にて対応しました。こういう意図であっていますか？
また、おそらく他に出している PR でも同様の要求があると思うのですが、

- 基本的には1つの作業に対して 1つの PR で行いたい
- こういう話をし始めると作業の終わりにキリがない
- `#@since` に対応する `#@end` を見つけるのが相当つらい

と、いう理由で他の PR での対応は一旦見送ろうかと考えているのですがいかがでしょか。

Reply:
今回は消せそうな `#@end` のところに `#@end` が追加されていたのが気になってしまったというのが大きかったので、その方針で大丈夫です。

Reply:
了解しました。ありがとうございます！